### PR TITLE
chore: prepare new dev version

### PR DESCRIPTION
**Pull request by Ad96el**
## Checklist:

- [ ] I have verified that the code works
  - [ ] No panics! (checked arithmetic ops, no indexing `array[3]` use `get(3)`, ...)
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ homepage = "https://kilt.io/"
 license-file = "LICENSE"
 readme = "README.md"
 repository = "https://github.com/KILTprotocol/kilt-node"
-version = "1.12.0-dev"
+version = "1.13.0-dev"
```
Comment (ntn-x2) [inline Cargo.toml:9]: Also the spec version need to be bumped to 11300 so that type augmentation works properly.

Comment (Ad96el) [reply Cargo.toml:9]: Yes. you are right: [1af49a7](https://github.com/KILTprotocol/kilt-node/pull/579/commits/1af49a7b5569bab0799fb24fee9a4757df2533c4)

**Comment by Ad96el** [comment]
I have also removed the migrations for the parachainStaking and the bump in storage versions. A minimal migration struct is still present, just in case we want to add a new pallet to the runtime and need to initialize the storage version.

File: Cargo.toml (hunk 1 above)
Comment (ntn-x2) [reply Cargo.toml:9]: Can you do it also for the DIP runtime templates?

Comment (Ad96el) [reply Cargo.toml:9]: [0ebdac0](https://github.com/KILTprotocol/kilt-node/pull/579/commits/0ebdac0b8f5335b20f89cf201405ef1ad890c2ef)

Comment (ntn-x2) [reply Cargo.toml:9]: What's with all these changes? 😅

Comment (Ad96el) [reply Cargo.toml:9]: rebased: [e0c4a3d](https://github.com/KILTprotocol/kilt-node/pull/579/commits/e0c4a3d6747b54641ab25c43afbb6b248ebfd2f5)

**Review by ntn-x2 — APPROVED** [review]

**Pull request merged**